site/src/pages/components/accordion.explainer.mdx

```diff
@@ -144,6 +146,24 @@ violate the exclusivity semantics?
 (This applies both to insertion by the parser
 and insertion through other methods like setting `innerHTML`.)
 
+### Printing
+
+For many accordions, it is desirable for the print layout of the page
+to have all items opened.
+This is currently not possible to do from a style sheet,
+but it probably should be possible.
+The presence of exclusive accordion semantics make this more difficult,
+because of the rule that only one item can be open.
+
```

Comment:
Worth a sentence or two also here that sometime's that's *not* what people want? I.e. sometimes (often?) when people hit print, they want to print what they're seeing, which would be just one open option. And perhaps that leads to this being a JS-driven thing during printing, where if the desire is to print "all", then the JS removes the `name` attribute and adds the `open` attribute to all `<details>`?